back/app/policies/project_folders/folder_policy.rb

```diff
@@ -25,7 +25,8 @@ def published_folders
 
     def show?
       return true if user && UserRoleService.new.can_moderate?(record, user)
-      return false if %w[published archived].exclude?(record.admin_publication.publication_status)
+      return false if record.admin_publication.publication_status == 'draft'
+      return true if record.projects.empty?
```

Comment:
So, it means all not draft folders without projects are visible to all users (not only visitors)?

Reply:
Yes

Reply:
Note that we don't have permissions on folders (i.e. no group permisions). So we filter such rights based on what projects a folder contains. Except now, if folder empty, anyone can see.